examples/tests/__init__.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
```

Comment:
tests should probably not live inside a folder in examples. Not sure where they should live, (either temporarily or in the long run). Maybe @kne42 or @AhmetCanSolak want's to weigh in

Reply:
will move once i sync up with loic's changes